lms/djangoapps/courseware/testutils.py

```diff
@@ -155,9 +155,9 @@ def verify_response(self, expected_response_code=200, url_params=None):
         return response
 
     @ddt.data(
-        ('vertical_block', ModuleStoreEnum.Type.mongo, 13),
+        ('vertical_block', ModuleStoreEnum.Type.mongo, 14),
```

Comment:
Could you explain why this change was necessary?

Reply:
When I had to invoke `PartitionService`, that would call `modulestore().get_course()`. Because of this call, it would make an extra call into the mongo.